src/components/MasterItems.js

```diff
@@ -49,11 +42,22 @@ class MasterItems extends Component {
     this.setState({ entries: [] });
   }
 
+  addComment(entryIndex, e) {
+    const updatedEntry = this.state.entries[entryIndex];
+    updatedEntry.comment = e.target.value;
```

Comment:
You're able to modify props on `updatedEntry` even though it's a `const`? Or does that only mean you can't reassign to that variable?